packages/yoga/src/Card/native/Card/Content.jsx

```diff
@@ -0,0 +1,7 @@
+import styled from 'styled-components';
```

Comment:
Is it necessary to have this Content component in this moment? 'couse it is empty (with no style).